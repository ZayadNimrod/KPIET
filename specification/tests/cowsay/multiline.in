Legi, Patres colendissimi, in Arabum monumentis, interrogatum Abdalam 1 Sarracenum, quid in hac quasi mundana scaena admirandum maxime spectaretur, nihil spectari homine admirabilius respondisse.

