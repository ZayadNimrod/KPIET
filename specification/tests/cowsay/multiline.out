 _________________________________________
/ Legi, Patres colendissimi, in Arabum mo \
| numentis, interrogatum Abdalam 1 Sarrac |
| enum, quid in hac quasi mundana scaena  |
| admirandum maxime spectaretur, nihil sp |
\ ectari homine admirabilius respondisse. /
 -----------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
